appendix/backup-and-restore/migrate-hosts.rst

```diff
@@ -0,0 +1,215 @@
+Migrate Zammad to new host
+**************************
+
+🙅‍♀️ This is a proof of concept, not a full how to
```

Comment:
🙅‍♀️ This is a proof of concept, not a full how to. It depends on your locale environment.